pnpm-lock.yaml

```diff
@@ -28,11 +28,11 @@ importers:
         specifier: workspace:*
         version: link:internal/jest-config
       '@types/node':
-        specifier: ^16.0.0
-        version: 16.18.12
+        specifier: ^18.0.0
+        version: 18.0.0
```

Comment:
if we want to specify a version, we could use `v18.18.2` here as well

Reply:
Yep! Good call, I'll update it to match the version in `.nvmrc`.